client/dvm/rpc/src/eth.rs

```diff
@@ -484,10 +484,7 @@ where
 				.get(&schema)
 				.unwrap_or(&self.fallback)
 				.storage_at(&id, address, index)
-				.ok_or(internal_err(
-					"Fetching account storage through override failed",
-				))?
-				.into());
+				.unwrap_or_default());
```

Comment:
It would be better to apply this change to the frontier.

Reply:
Is frontier has a pending PR waiting for being merged?

Please add some document here with the link to related issue or PR.

Reply:
Not yet until now. let me do a fix about this issue.

Reply:
https://github.com/paritytech/frontier/pull/316